packages/reassure-cli/src/commands/git.ts

```diff
@@ -27,7 +28,7 @@ export async function getGitCommitHash() {
     const commitHash = await git.revparse(['HEAD']);
     return commitHash.trim() ? commitHash : undefined;
   } catch (error) {
-    console.log('Failed to get git commit hash', error);
+    printError(logLevel, 'Failed to get git commit hash', error);
```

Comment:
Can we rather have a logger module that will have level state, so that we do not have to pass logLevel everywhere?

Reply:
done